command/agent/command.go

```diff
@@ -1549,6 +1562,11 @@ Client Options:
 
   -network-interface
     Forces the network fingerprinter to use the specified network interface.
+  
+  -preferred-ip-family
+    Specify which IP family to prefer when selecting an IP address of the
+	network interface. Valid values are "ipv4" and "ipv6". When not specified,
+	the agent will not sort the addresses and use the first one.
```

Comment:
this doesn't match the actual flag code, also some whitespace inconsistency:

```suggestion
  -preferred-address-family
    Specify which IP family to prefer when selecting an IP address of the
    network interface. Valid values are "ipv4" and "ipv6". When not specified,
    the agent will not sort the addresses and use the first one.
```